pkg/api/v1alpha1/cluster_webhook.go

```diff
@@ -54,19 +55,33 @@ var _ webhook.Validator = &Cluster{}
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (r *Cluster) ValidateCreate() error {
 	clusterlog.Info("validate create", "name", r.Name)
-	if r.IsReconcilePaused() {
-		clusterlog.Info("cluster is paused, so allowing create", "name", r.Name)
-		return nil
-	}
+	var allErrs []error
+
 	if !features.IsActive(features.FullLifecycleAPI()) {
-		return apierrors.NewBadRequest("Creating new cluster on existing cluster is not supported")
+		allErrs = append(allErrs,
+			fmt.Errorf("creating new cluster on existing cluster is not supported"))
 	}
+
 	if r.IsSelfManaged() {
-		return apierrors.NewBadRequest("Creating new cluster on existing cluster is not supported")
+		allErrs = append(
+			allErrs,
+			fmt.Errorf("creating new cluster on existing cluster is not supported for self managed clusters"))
+	}
+
+	if err := r.Validate(); err != nil {
+		allErrs = append(
+			allErrs,
+			fmt.Errorf("cluster validation failed %v", err.Error()))
```

Comment:
```suggestion
			fmt.Errorf("invalid cluster %v", err.Error()))
```

maybe we should use some constructor for invalid field?